pkg/compliance/evaluator_xccdf.go

```diff
@@ -364,3 +364,37 @@ func evaluateXCCDFRule(ctx context.Context, hostname string, statsdClient *stats
 
 	return events
 }
+
+// Finish an XCCDF benchmark by terminating the oscap-io processes.
+func FinishXCCDFBenchmark(ctx context.Context, benchmark *Benchmark) {
+	if len(oscapIOs) == 0 {
```

Comment:
Shouldn't it be done after taking the oscapIOsMu lock ?

Reply:
Indeed.